### PR TITLE
add error logging and send first data as not headers

### PR DESCRIPTION
**Pull request by skothari-tibco**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,7 @@ module github.com/project-flogo/stream
 
 require (
 	github.com/project-flogo/core v0.9.4
+	github.com/project-flogo/stream/trigger/streamtester v0.0.0-20191202150253-ad7163ed9e34 // indirect
```
Comment (fm-tibco) [inline go.mod]: should be removed

File: trigger/streamtester/timer.go [hunk 2]
```diff
@@ -148,25 +148,29 @@ func (t *Trigger) start(handler trigger.Handler, settings *HandlerSettings, emit
...
 			}
 			triggerData = prepareRepeatingData(dataTemp, emitInfo, settings.Header)
 
 			emitInfo.Count = emitInfo.Count + 1
 			//triggerData.Data = dataTemp
 		}
-		
-		t.logger.Debug("Data passed to Handler..",triggerData )
+
+		t.logger.Debug("Data passed to Handler..", triggerData.Data)
```
Comment (fm-tibco) [inline trigger/streamtester/timer.go]: This seems less helpful than just triggerData, consider:

t.logger.Debug("Trigger data: %#v", triggerData)

File: trigger/streamtester/timer.go [hunk 3]
```diff
@@ -256,7 +257,10 @@ func ReadCsvInterval(path string, emitInfo *HandlerEmitterInfo) ([]string, error
 			return nil, err
 		}
 		emitInfo.Lines = data
-
+		defer func() {
```
Comment (fm-tibco) [inline trigger/streamtester/timer.go]: Let me under stand this, you are returning the header row, but increasing the count by 1.  What is count?  Maybe you should rename count to something more informative.  maybe "Index" or "RowNum" or something if thats what it is.

Comment (skothari-tibco) [reply trigger/streamtester/timer.go]: Changed it to CurrentIndex for better clear understanding.

**Pull request merged**